vllm/v1/structured_output/__init__.py

```diff
@@ -85,7 +86,7 @@ def _delayed_init(self):
         self.executor = ThreadPoolExecutor(max_workers=max_workers)
         self._grammar_bitmask = xgr.allocate_token_bitmask(
             self.vllm_config.scheduler_config.max_num_seqs,
-            self.vocab_size,
+            self.mask_size,
```

Comment:
Is the `bitmask` applied to the logits?

Reply:
correct, applied here: https://github.com/vllm-project/vllm/blob/270a5da495d24e947a71e2fa0c56635f4fad2dc3/vllm/v1/worker/gpu_model_runner.py#L869-L914

Reply:
It would make sense to me that this is the vocab_size then, since then it matches the shape of the logits